frontend/src/app/page.tsx

```diff
@@ -1,13 +1,275 @@
+                <Image
+                  src="/images/feature-hero-image@3x.png"
+                  width={387}
+                  height={215}
+                  alt=""
+                  className="hidden md:block mb-8"
+                />
+              )}
+            </div>
+            <div className="px-4 md:px-0 md:w-[80%] mx-auto">
+              <p className="font-medium text-base md:text-[20px] md:ont-semibold">
+                Compare Payment Services
+              </p>
+              <p className="leading-5 md:text-base mt-2 mb-1 text-xs">
+                Databases are foundational components of software applications,
+                used to store, manage, and retrieve data. They come in two
+                primary types: relational (SQL) and non-relational (NoSQL)
+                databases.
+              </p>
+            </div>
+          </div>
+        ))}
+      </div>
+      <button className="md:mt-[45px] mt-10 md:px-10 px-6 py-2 bg-primary text-white rounded-full font-lora font-semibold text-sm md:text-xl">
+        Explore All Features
+      </button>
+    </div>
+  );
+};
+
+const AllFeaturesSection = () => {
+  return (
+    <div className="bg-[#D9D9D933] py-12 max-w-[1286px] grid md:grid-cols-3 rounded-[20px] mx-auto">
+      {allFeatures.map((feature, index) => (
+        <div
+          key={index}
+          className={`md:border-l ${index < 5 ? 'border-b md:border-b-0' : ''} ${
+            index < 3 ? 'md:border-b' : ''
+          } border-[#2029381F] md:px-10 py-4 px-8`}
+        >
+          <div className="flex items-center gap-2">
+            <GoDotFill />
+            <p className="md:text-xl text-sm font-medium">{feature.title}</p>
+          </div>
+          <p className="md:leading-[25px] leading-5 text-xs mt-[15px]">
+            {feature.description}
+          </p>
+        </div>
+      ))}
+    </div>
+  );
+};
+
+const FaQSection = ({ id }: { id?: string }) => {
+  return (
+    <div id={id}>
+      <div className="flex flex-col items-center">
+        <button className="md:px-10 md:mt-[93px] mt-[50px] px-6 py-1.5 text-base font-roboto rounded-md border border-[#202938BF]">
+          FAQS
+        </button>
+        <h1 className="text-center font-bold text-primary md:text-[45px] md:leading-[60px] leading-7 my-[20px] text-[20px]">
+          Everything you <br />{' '}
+          <span className="text-[#8D9CB3]">Need to Know</span>
+        </h1>
+        <p className="mt-4 text-center max-w-[645px] text-primary">
+          Innovative and impactful, our features are designed to set new
+          standards and drive exceptional results. Experience the difference
+          that cutting-edge technology and unique solutions can make.
+        </p>
+      </div>
+      <div>
+        {faqs.map((faq, index) => (
+          <AccordionItem key={index} title={faq.title} content={faq.content} />
+        ))}
+      </div>
+    </div>
+  );
+};
+
+const navigationLinks = [
```

Comment:
Create a folder name utils/constants.ts and move this in there

Reply:
resolved